### PR TITLE
Add theme changing option in main menu

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/1165
[Problem] No option to change theme in UIComponents app
[Solution] Add option for theme modification on main page

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
image

File: examples/mobile/UIComponents/js/app.js [hunk 1]
```diff
@@ -19,4 +21,19 @@
...
+		themeChanger.addEventListener("change", function (event) {
+			tau.theme.setTheme(event.target.value);
+		});
+
+		themeChangerButton.addEventListener("click", function () {
+			var dropDownMenuWidget = tau.widget.DropdownMenu(themeChanger);
+
+			dropDownMenuWidget.open();
+		});
+	});
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/js/app.js:42]: Need to remove the event listeners when "pagebeforehide".

Comment (lslachciak) [reply examples/mobile/UIComponents/js/app.js:42]: done

**Comment by lslachciak** [comment]
Improved PR according to the @HunseopJeong comment

**Review by HunseopJeong — APPROVED** [review]
Looks good to me

**Pull request merged**